src/components/deck-collection-filters/exp-cost-filter.tsx

```diff
@@ -0,0 +1,68 @@
+// Currently unused, functionality preserved for 'My Decks' dedicated page.
```

Comment:
Nothing for right now, I wonder if we should accommodate the standalone format with 9/19/29 XP here 🤔